### PR TITLE
*: correct TiDB tool names

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? 

This PR corrects the tool names:

+ `TiDB-Ansible` -> `TiDB Ansible`
+ `TiDB operator` or `TiDB-Operator` -> `TiDB Operator`
+ `mydumper` or `MyDumper` -> `Mydumper`
+ `TiDB-Lightning` or `Lightning` -> `TiDB Lightning`
+ `TiDB-Binlog` -> `TiDB Binlog`

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs/pull/1466

### Which version does your change affect? 

`dev`, `v2.1`, `v3.0`

### CODE REVIEW

File: v2.1/how-to/deploy/orchestrated/ansible.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-title: TiDB-Ansible 部署方案
+title:  部署方案
```
Comment (yikeke) [inline v2.1/how-to/deploy/orchestrated/ansible.md]:
```suggestion
title:  部署方案
```
Seems that you didn't replace `TiDB-Ansible` successfully in this file. Please replace all of them again.

File: v2.1/how-to/deploy/orchestrated/offline-ansible.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
 ---
-title: 离线 TiDB-Ansible 部署方案
+title: 离线  部署方案
```
Comment (yikeke) [inline v2.1/how-to/deploy/orchestrated/offline-ansible.md]:
```suggestion
title: 离线 TiDB Ansible 部署方案
```
ditto

File: v2.1/how-to/get-started/tispark.md [hunk 3]
```diff
@@ -5,13 +5,13 @@ category: how-to
 
 # TiSpark 快速入门指南
 
-为了让大家快速体验 [TiSpark](/reference/tispark.md)，通过 TiDB-Ansible 安装的 TiDB 集群中默认已集成 Spark、TiSpark jar 包及 TiSpark sample data。
+为了让大家快速体验 [TiSpark](/reference/tispark.md)，通过  安装的 TiDB 集群中默认已集成 Spark、TiSpark jar 包及 TiSpark sample data。
```
Comment (yikeke) [inline v2.1/how-to/get-started/tispark.md]: ditto

File: v2.1/how-to/maintain/ansible-operations.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
 ---
-title: TiDB-Ansible 常见运维操作
+title:  常见运维操作
```
Comment (yikeke) [inline v2.1/how-to/maintain/ansible-operations.md]: ditto

File: v2.1/how-to/scale/with-ansible.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
 ---
-title: 使用 TiDB-Ansible 扩容缩容 TiDB 集群
+title: 使用  扩容缩容 TiDB 集群
```
Comment (yikeke) [inline v2.1/how-to/scale/with-ansible.md]: ditto

File: v2.1/how-to/upgrade/from-previous-version.md [hunk 6]
```diff
@@ -26,7 +26,7 @@ aliases: ['/docs-cn/v2.1/how-to/upgrade/to-tidb-3.0/']
 >
 > 如果已经安装了 Ansible 及其依赖，可跳过该步骤。
 
-TiDB-Ansible release-3.0 版本依赖 Ansible 2.4.2 及以上版本（`ansible>=2.4.2`），另依赖 Python 模块：`jinja2>=2.9.6` 和 `jmespath>=0.9.0`。为方便管理依赖，建议使用 `pip` 安装 Ansible 及其依赖，可参照[在中控机器上安装 Ansible 及其依赖](/how-to/deploy/orchestrated/ansible.md#在中控机器上安装-ansible-及其依赖) 进行安装。离线环境参照[在中控机器上离线安装 Ansible 及其依赖](/how-to/deploy/orchestrated/offline-ansible.md#在中控机器上离线安装-ansible-及其依赖)。
+ release-3.0 版本依赖 Ansible 2.4.2 及以上版本（`ansible>=2.4.2`），另依赖 Python 模块：`jinja2>=2.9.6` 和 `jmespath>=0.9.0`。为方便管理依赖，建议使用 `pip` 安装 Ansible 及其依赖，可参照[在中控机器上安装 Ansible 及其依赖](/how-to/deploy/orchestrated/ansible.md#在中控机器上安装-ansible-及其依赖) 进行安装。离线环境参照[在中控机器上离线安装 Ansible 及其依赖](/how-to/deploy/orchestrated/offline-ansible.md#在中控机器上离线安装-ansible-及其依赖)。
```
Comment (yikeke) [inline v2.1/how-to/upgrade/from-previous-version.md]: ditto

File: v2.1/how-to/upgrade/rolling-updates-with-ansible.md [hunk 7]
```diff
@@ -1,9 +1,9 @@
 ---
-title: 使用 TiDB-Ansible 升级 TiDB 集群
+title: 使用  升级 TiDB 集群
```
Comment (yikeke) [inline v2.1/how-to/upgrade/rolling-updates-with-ansible.md]: ditto

File: v2.1/how-to/upgrade/to-tidb-2.1.md [hunk 8]
```diff
@@ -21,7 +21,7 @@ category: how-to
 
 ## 在中控机器上安装 Ansible 及其依赖
 
-TiDB-Ansible release-2.1 版本依赖 2.4.2 及以上但不高于 2.7.0 的 Ansible 版本（`ansible>=2.4.2,<2.7.0`），另依赖 Python 模块：`jinja2>=2.9.6` 和 `jmespath>=0.9.0`。为方便管理依赖，新版本使用 `pip` 安装 Ansible 及其依赖，可参照[在中控机器上安装 Ansible 及其依赖](/how-to/deploy/orchestrated/ansible.md#在中控机器上安装-ansible-及其依赖) 进行安装。离线环境参照[在中控机器上离线安装 Ansible 及其依赖](/how-to/deploy/orchestrated/offline-ansible.md#在中控机器上离线安装-ansible-及其依赖)。
+ release-2.1 版本依赖 2.4.2 及以上但不高于 2.7.0 的 Ansible 版本（`ansible>=2.4.2,<2.7.0`），另依赖 Python 模块：`jinja2>=2.9.6` 和 `jmespath>=0.9.0`。为方便管理依赖，新版本使用 `pip` 安装 Ansible 及其依赖，可参照[在中控机器上安装 Ansible 及其依赖](/how-to/deploy/orchestrated/ansible.md#在中控机器上安装-ansible-及其依赖) 进行安装。离线环境参照[在中控机器上离线安装 Ansible 及其依赖](/how-to/deploy/orchestrated/offline-ansible.md#在中控机器上离线安装-ansible-及其依赖)。
```
Comment (yikeke) [inline v2.1/how-to/upgrade/to-tidb-2.1.md]: ditto

File: v2.1/reference/tools/syncer.md [hunk 9]
```diff
@@ -13,15 +13,15 @@ Syncer 包含在 tidb-enterprise-tools 安装包中，可[在此下载](/referen
...
+![Syncer 架构](/media/syncer-architecture.png)
 
 ## Syncer 部署位置
 
 Syncer 可以部署在任一台可以连通对应的 MySQL 和 TiDB 集群的机器上，推荐部署在 TiDB 集群。
 
 ## Syncer 增量导入数据示例
 
-使用前请详细阅读 [Syncer 同步前预检查](#syncer-同步前检查)。
+使用前请详细阅读 [Syncer 同步前预检查](#Syncer-同步前检查)。
```
Comment (yikeke) [inline v2.1/reference/tools/syncer.md]:
```suggestion
使用前请详细阅读 [Syncer 同步前预检查](#syncer-同步前检查)。
```
No need to update the link.

File: v2.1/releases/2.0.10.md [hunk 10]
```diff
@@ -5,7 +5,7 @@ category: Releases
 
 # TiDB 2.0.10 Release Notes
 
-2018 年 12 月 18 日，TiDB 发布 2.0.10 版，TiDB-Ansible 相应发布 2.0.10 版本。该版本在 2.0.9 版的基础上，对系统兼容性、稳定性做出了改进。
+2018 年 12 月 18 日，TiDB 发布 2.0.10 版， 相应发布 2.0.10 版本。该版本在 2.0.9 版的基础上，对系统兼容性、稳定性做出了改进。
```
Comment (yikeke) [inline v2.1/releases/2.0.10.md]:
```suggestion
2018 年 12 月 18 日，TiDB 发布 2.0.10 版，TiDB Ansible 相应发布 2.0.10 版本。该版本在 2.0.9 版的基础上，对系统兼容性、稳定性做出了改进。
```

File: v2.1/releases/2.0.11.md [hunk 11]
```diff
@@ -5,7 +5,7 @@ category: Releases
 
 # TiDB 2.0.11 Release Notes
 
-2019 年 1 月 3 日，TiDB 发布 2.0.11 版，TiDB-Ansible 相应发布 2.0.11 版本。该版本在 2.0.10 版的基础上，对系统兼容性、稳定性做出了改进。
+2019 年 1 月 3 日，TiDB 发布 2.0.11 版， 相应发布 2.0.11 版本。该版本在 2.0.10 版的基础上，对系统兼容性、稳定性做出了改进。
```
Comment (yikeke) [inline v2.1/releases/2.0.11.md]: ditto

File: v2.1/releases/2.1.2.md [hunk 12]
```diff
@@ -5,11 +5,11 @@ category: Releases
 
 # TiDB 2.1.2 Release Notes
 
-2018 年 12 月 22 日，TiDB 发布 2.1.2 版，TiDB-Ansible 相应发布 2.1.2 版本。该版本在 2.1.1 版的基础上，对系统兼容性、稳定性做出了改进。
+2018 年 12 月 22 日，TiDB 发布 2.1.2 版， 相应发布 2.1.2 版本。该版本在 2.1.1 版的基础上，对系统兼容性、稳定性做出了改进。
```
Comment (yikeke) [inline v2.1/releases/2.1.2.md]: ditto

File: v2.1/releases/2.1.3.md [hunk 13]
```diff
@@ -5,7 +5,7 @@ category: Releases
 
 # TiDB 2.1.3 Release Notes
 
-2019 年 01 月 28 日，TiDB 发布 2.1.3 版，TiDB-Ansible 相应发布 2.1.3 版本。相比 2.1.2 版本，该版本对系统稳定性、优化器、统计信息以及执行引擎做了很多改进。
+2019 年 01 月 28 日，TiDB 发布 2.1.3 版， 相应发布 2.1.3 版本。相比 2.1.2 版本，该版本对系统稳定性、优化器、统计信息以及执行引擎做了很多改进。
```
Comment (yikeke) [inline v2.1/releases/2.1.3.md]: ditto

File: v2.1/releases/2.1.4.md [hunk 14]
```diff
@@ -5,7 +5,7 @@ category: Releases
 
 # TiDB 2.1.4 Release Notes
 
-2019 年 2 月 15 日，TiDB 发布 2.1.4 版，TiDB-Ansible 相应发布 2.1.4 版本。相比 2.1.3 版本，该版本对系统稳定性、优化器、统计信息以及执行引擎做了很多改进。
+2019 年 2 月 15 日，TiDB 发布 2.1.4 版， 相应发布 2.1.4 版本。相比 2.1.3 版本，该版本对系统稳定性、优化器、统计信息以及执行引擎做了很多改进。
```
Comment (yikeke) [inline v2.1/releases/2.1.4.md]: ditto

File: v2.1/releases/2.1.5.md [hunk 15]
```diff
@@ -5,7 +5,7 @@ category: Releases
 
 # TiDB 2.1.5 Release Notes
 
-2019 年 2 月 28 日，TiDB 发布 2.1.5 版，TiDB-Ansible 相应发布 2.1.5 版本。相比 2.1.4 版本，该版本对系统稳定性、优化器、统计信息以及执行引擎做了很多改进。
+2019 年 2 月 28 日，TiDB 发布 2.1.5 版， 相应发布 2.1.5 版本。相比 2.1.4 版本，该版本对系统稳定性、优化器、统计信息以及执行引擎做了很多改进。
```
Comment (yikeke) [inline v2.1/releases/2.1.5.md]: ditto

File: v2.1/releases/2.1.6.md [hunk 16]
```diff
@@ -5,7 +5,7 @@ category: Releases
 
 # TiDB 2.1.6 Release Notes
 
-2019 年 3 月 15 日，TiDB 发布 2.1.6 版，TiDB-Ansible 相应发布 2.1.6 版本。相比 2.1.5 版本，该版本对系统稳定性、优化器、统计信息以及执行引擎做了很多改进。
+2019 年 3 月 15 日，TiDB 发布 2.1.6 版， 相应发布 2.1.6 版本。相比 2.1.5 版本，该版本对系统稳定性、优化器、统计信息以及执行引擎做了很多改进。
```
Comment (yikeke) [inline v2.1/releases/2.1.6.md]: ditto

File: v2.1/releases/2.1.7.md [hunk 17]
```diff
@@ -9,7 +9,7 @@ category: Releases
 
 TiDB 版本：2.1.7
 
-TiDB-Ansible 版本：2.1.7
+ 版本：2.1.7
```
Comment (yikeke) [inline v2.1/releases/2.1.7.md]: ditto

File: v2.1/releases/2.1.8.md [hunk 18]
```diff
@@ -9,7 +9,7 @@ category: Releases
 
 TiDB 版本：2.1.8
 
-TiDB-Ansible 版本：2.1.8
+ 版本：2.1.8
```
Comment (yikeke) [inline v2.1/releases/2.1.8.md]: ditto

File: v2.1/releases/2.1.9.md [hunk 19]
```diff
@@ -9,7 +9,7 @@ category: Releases
 
 TiDB 版本：2.1.9
 
-TiDB-Ansible 版本：2.1.9
+ 版本：2.1.9
```
Comment (yikeke) [inline v2.1/releases/2.1.9.md]: ditto

File: v2.1/releases/3.0.0-beta.1.md [hunk 20]
```diff
@@ -9,11 +9,11 @@ category: Releases
 
 TiDB 版本：3.0.0-beta.1
 
-TiDB-Ansible 版本：3.0.0-beta.1
+ 版本：3.0.0-beta.1
```
Comment (yikeke) [inline v2.1/releases/3.0.0-beta.1.md]: ditto

File: v2.1/releases/3.0.0-rc.1.md [hunk 21]
```diff
@@ -9,11 +9,11 @@ category: Releases
 
 TiDB 版本：3.0.0-rc.1
 
-TiDB-Ansible 版本：3.0.0-rc.1
+ 版本：3.0.0-rc.1
```
Comment (yikeke) [inline v2.1/releases/3.0.0-rc.1.md]: ditto

File: v2.1/releases/3.0beta.md [hunk 22]
```diff
@@ -5,7 +5,7 @@ category: Releases
 
 # TiDB 3.0 Beta Release Notes
 
-2019 年 1 月 19 日，TiDB 发布 3.0 Beta 版，TiDB-Ansible 相应发布 3.0 Beta 版本。相比 2.1 版本，该版本对系统稳定性、优化器、统计信息以及执行引擎做了很多改进。
+2019 年 1 月 19 日，TiDB 发布 3.0 Beta 版， 相应发布 3.0 Beta 版本。相比 2.1 版本，该版本对系统稳定性、优化器、统计信息以及执行引擎做了很多改进。
```
Comment (yikeke) [inline v2.1/releases/3.0beta.md]: ditto

File: v2.1/tispark/tispark-quick-start-guide_v1.x.md [hunk 23]
```diff
@@ -5,7 +5,7 @@ category: tispark
 
 # TiSpark 快速入门指南
 
-为了让大家快速体验 [TiSpark](/reference/tispark.md)，通过 TiDB-Ansible 安装的 TiDB 集群中默认已集成 Spark、TiSpark jar 包及 TiSpark sample data。
+为了让大家快速体验 [TiSpark](/reference/tispark.md)，通过  安装的 TiDB 集群中默认已集成 Spark、TiSpark jar 包及 TiSpark sample data。
```
Comment (yikeke) [inline v2.1/tispark/tispark-quick-start-guide_v1.x.md]: ditto

File: v3.0/faq/tidb-lightning.md [hunk 24]
```diff
@@ -78,11 +78,11 @@ sql-mode = ""
 
 ## 如何正确关闭 `tikv-importer` 进程？
 
-如使用 TiDB-Ansible 部署，在 Importer 的服务器上运行 `scripts/stop_importer.sh` 即可。否则，可通过 `ps aux | grep tikv-importer` 获取进程ID，然后 `kill «pid»`。
+如使用  部署，在 Importer 的服务器上运行 `scripts/stop_importer.sh` 即可。否则，可通过 `ps aux | grep tikv-importer` 获取进程ID，然后 `kill «pid»`。
```
Comment (yikeke) [inline v3.0/faq/tidb-lightning.md]: ditto

File: v3.0/faq/tidb.md [hunk 25]
```diff
@@ -236,7 +236,7 @@ TiDB 支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器
 | **变量** | **含义** |
 | --- | --- |
 | cluster_name | 集群名称，可调整 |
-| tidb_version | TiDB 版本，TiDB-Ansible 各分支默认已配置 |
+| tidb_version | TiDB 版本， 各分支默认已配置 |
```
Comment (yikeke) [inline v3.0/faq/tidb.md]: ditto

File: v3.0/faq/tidb.md [hunk 26]
```diff
@@ -759,7 +759,7 @@ sqoop export \
 
 - job_name: &#39;syncer_ops&#39; // 任务名字
     static_configs:
-- targets: [&#39;10.10.1.1:10096&#39;] //syncer监听地址与端口，通知 prometheus 拉取 syncer 的数据。
+- targets: [&#39;10.10.1.1:10096&#39;] //Syncer 监听地址与端口，通知 prometheus 拉取 Syncer 的数据。
```
Comment (yikeke) [inline v3.0/faq/tidb.md]:
```suggestion
- targets: [&#39;10.10.1.1:10096&#39;] // Syncer 监听地址与端口，通知 prometheus 拉取 Syncer 的数据。
```

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 27]
```diff
@@ -1,14 +1,14 @@
 ---
-title: TiDB-Ansible 部署方案
+title:  部署方案
```
Comment (yikeke) [inline v3.0/how-to/deploy/orchestrated/ansible.md]: The `v3.0` files may have the same problem as the `dev` files. Please check whether `TiDB-Ansible` is replaced correctly in all changed files. @TomShawn

File: v2.1/how-to/deploy/orchestrated/ansible.md [hunk 28]
```diff
@@ -1,13 +1,13 @@
...
 category: how-to
 ---
 
-# TiDB-Ansible 部署方案
+# 部署方案
```
Comment (lilin90) [inline v2.1/how-to/deploy/orchestrated/ansible.md]: Why remove TiDB Ansible here? Suggested version: **使用 TiDB Ansible 部署 TiDB 集群**

File: dev/reference/tools/syncer.md [hunk 29]
```diff
@@ -13,15 +13,15 @@ Syncer 包含在 tidb-enterprise-tools 安装包中，可[在此下载](/referen
...
+![Syncer 架构](/media/syncer-architecture.png)
 
 ## Syncer 部署位置
 
 Syncer 可以部署在任一台可以连通对应的 MySQL 和 TiDB 集群的机器上，推荐部署在 TiDB 集群。
 
 ## Syncer 增量导入数据示例
 
-使用前请详细阅读 [Syncer 同步前预检查](#syncer-同步前检查)。
+使用前请详细阅读 [Syncer 同步前预检查](#Syncer-同步前检查)。
```
Comment (lilin90) [inline dev/reference/tools/syncer.md]: It must be lower-case in such links.

```suggestion
使用前请详细阅读 [Syncer 同步前预检查](#syncer-同步前检查)。
```

File: v2.1/faq/tidb.md [hunk 30]
```diff
@@ -782,7 +782,7 @@ sqoop export \
 
 ##### 4.2.1.6 使用 Syncer gtid 的方式同步时，同步过程中会不断更新 syncer.meta 文件，如果 Syncer 所在的机器坏了，导致 syncer.meta 文件所在的目录丢失，该如何处理？
 
-当前 Syncer 版本的没有进行高可用设计，Syncer 目前的配置信息 syncer.meta 直接存储在硬盘上，其存储方式类似于其他 MySQL 生态工具，比如 mydumper。 因此，要解决这个问题当前可以有两个方法：
+当前 Syncer 版本的没有进行高可用设计，Syncer 目前的配置信息 syncer.meta 直接存储在硬盘上，其存储方式类似于其他 MySQL 生态工具，比如 Mydumper。 因此，要解决这个问题当前可以有两个方法：
```
Comment (lilin90) [inline v2.1/faq/tidb.md]:
```suggestion
当前 Syncer 版本的没有进行高可用设计，Syncer 目前的配置信息 syncer.meta 直接存储在硬盘上，其存储方式类似于其他 MySQL 生态工具，比如 Mydumper。因此，要解决这个问题当前可以有两个方法：
```

File: v3.0/faq/tidb.md [hunk 31]
```diff
@@ -211,7 +211,7 @@ TiDB 支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器
 
 #### 2.2.1 Ansible 部署方式（强烈推荐）
 
-详细可参考 [TiDB Ansible 部署方案](/how-to/deploy/orchestrated/ansible.md)。
+详细可参考 [使用 TiDB Ansible 部署 TiDB 集群](/how-to/deploy/orchestrated/ansible.md)。
```
Comment (lilin90) [inline v3.0/faq/tidb.md]:
```suggestion
详细可参考[使用 TiDB Ansible 部署 TiDB 集群](/how-to/deploy/orchestrated/ansible.md)。
```

File: dev/faq/tidb.md [hunk 32]
```diff
@@ -210,7 +210,7 @@ TiDB 支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器
 
 #### 2.2.1 Ansible 部署方式（强烈推荐）
 
-详细可参考 [TiDB Ansible 部署方案](/how-to/deploy/orchestrated/ansible.md)。
+详细可参考 [使用 TiDB Ansible 部署 TiDB 集群](/how-to/deploy/orchestrated/ansible.md)。
```
Comment (lilin90) [inline dev/faq/tidb.md]:
```suggestion
详细可参考[使用 TiDB Ansible 部署 TiDB 集群](/how-to/deploy/orchestrated/ansible.md)。
```

File: v3.0/reference/tools/syncer.md [hunk 33]
```diff
@@ -14,15 +14,15 @@ Syncer 包含在 tidb-enterprise-tools 安装包中，可[在此下载](/referen
...
+![Syncer 架构](/media/syncer-architecture.png)
 
 ## Syncer 部署位置
 
 Syncer 可以部署在任一台可以连通对应的 MySQL 和 TiDB 集群的机器上，推荐部署在 TiDB 集群。
 
 ## Syncer 增量导入数据示例
 
-使用前请详细阅读 [Syncer 同步前预检查](#syncer-同步前检查)。
+使用前请详细阅读 [Syncer 同步前预检查](#Syncer-同步前检查)。
```
Comment (lilin90) [inline v3.0/reference/tools/syncer.md]:
```suggestion
使用前请详细阅读 [Syncer 同步前预检查](#syncer-同步前检查)。
```

**Pull request merged**